features/google/src/popUps/info/base.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import infoPopUps from "./";
+import { SideSheet, SideSwiper } from "@polypoly-eu/poly-look";
+
+const BaseInfoPopUp = ({ onClose, name }) => {
+    return (
+        <SideSwiper
+            onClose={onClose}
+            open={true}
+            lastChildSelector=".poly-button"
+            Component={(props) => (
+                <SideSheet title={"How to read this"} okLabel={"ok"} {...props}>
```

Comment:
We're missing `className="poly-theme-light"` somewhere, I think it would go here.

Reply:
We could add it directly in `SideSheet`. That's what I've done on the fb PR.